crates/sui-config/tests/snapshot_tests.rs

```diff
@@ -108,7 +108,7 @@ fn network_config_snapshot_matches() {
             consensus_config
                 .narwhal_config
                 .prometheus_metrics
-                .socket_addr = fake_socket;
+                .socket_addr = Multiaddr::empty();
```

Comment:
Can I do that? I understand it is ok since it is only a test (and that we where using a fake socket before). This forced me to changed `crates/sui-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap` accordingly.

Reply:
This breaks because your Narwhal import brings in https://github.com/MystenLabs/narwhal/pull/625, itself a breaking configuration change. 

You can fix the test this way, though I would much prefer if you populated this line and the `consensus_config.consensus_address` above with a real dummy `MultiAddr` (à la "ip4/127.0.0.1/tcp/1234").

More importantly, you'll want to mark this as a breaking change (and help change configs where relevant).